ui/app/clinical/displaycontrols/treatmentData/directives/treatmentData.js

```diff
@@ -21,7 +21,7 @@ angular.module('bahmni.clinical')
 
                 var startDate = null, endDate = null, getEffectiveOrdersOnly = false;
                 if (programConfig.showDetailsWithinDateRange) {
-                    startDate = $stateParams.dateEnrolled;
+                    startDate = $stateParamqs.dateEnrolled;
```

Comment:
@buvaneswari-arun can you check is this `stateParamqs` or `stateParams` ?

Reply:
Modified